ISS-1378 Make pixee/upload-tool-results-action Sonar Host URL Consistent With Sonar Actions (#26)
pixee/upload-tool-results-action: README.md

  if: always() && steps.sonarqube-analysis.outcome == 'success'
  with:
    tool: sonar

>     sonar-host: ${{ vars.SONAR_HOST_URL }}/api

Reviewer: I think we should remove the /api and add it inside the action logic, so teh user only have to provide the host url

    # Required
    tool:

    # Token for authenticating requests to Sonar.
    # Required, when tool is "sonar" and "file" has not been set. Only required for private repository.
    sonar-token:

    # Key identifying the Sonar component to be analyzed. Only necessary if deviating from Sonar's established convention.
    # Default: `owner_repo`
    sonar-component-key:

>     # Base host of the Sonar or SonarQube API. Use this to switch from Sonar to SonarQube.

Reviewer 2: "Base host" is confusing. In this context, let's avoid the ambiguous "Sonar": we mean to differentiate between SonarCloud and SonarQube.

```suggestion
    # SonarCloud or SonarQube host URL. Use this to switch from SonarCloud to SonarQube.
```
